examples/benchmark/main.rs

```diff
@@ -0,0 +1,271 @@
+use std::{collections::HashMap, env, time::Instant};
+
+use clap::Parser;
+use futures::{stream::BoxStream, StreamExt};
+use num_bigint::BigUint;
+use tracing::info;
+use tracing_subscriber::EnvFilter;
+use tycho_simulation::{
+    evm::{
+        decoder::StreamDecodeError,
+        engine_db::tycho_db::PreCachedDB,
+        protocol::{
+            filters::{balancer_pool_filter, curve_pool_filter},
+            uniswap_v2::state::UniswapV2State,
+            uniswap_v3::state::UniswapV3State,
+            vm::state::EVMPoolState,
+        },
+        stream::ProtocolStreamBuilder,
+    },
+    models::Token,
+    protocol::models::BlockUpdate,
+    tycho_client::feed::component_tracker::ComponentFilter,
+    tycho_core::dto::Chain,
+    utils::load_all_tokens,
+};
+
+#[derive(Parser, Debug, Clone, PartialEq)]
+struct Cli {
+    /// The exchanges to benchmark
+    #[clap(long, number_of_values = 1)]
+    exchange: Vec<String>,
```

Comment:
could we limit this to the exchanges that are actually supported? I think we should say somewhere which exchanges are actually supported